response_play.go

```diff
@@ -0,0 +1,135 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/google/go-querystring/query"
+)
+
+type ResponsePlay struct {
+	ID                 string          `json:"id,omitempty"`
+	Type               string          `json:"type,omitempty"`
+	Summary            string          `json:"summary,omitempty"`
+	Self               string          `json:"self,omitempty"`
+	HTMLURL            string          `json:"html_url,omitempty"`
+	Name               string          `json:"name,omitempty"`
+	Description        string          `json:"description"`
+	Team               *APIReference   `json:"team,omitempty"`
+	Subscribers        []*APIReference `json:"subscribers,omitempty"`
+	SubscribersMessage string          `json:"subscribers_message"`
+	Responders         []*APIReference `json:"responders,omitempty"`
+	RespondersMessage  string          `json:"responders_message"`
+	Runnability        string          `json:"runnability"`
+	ConferenceNumber   string          `json:"conference_number"`
+	ConferenceUrl      string          `json:"conference_url"`
+	ConferenceType     string          `json:"conference_type"`
```

Comment:
So I think this PR is basically ready to go, but I do have some reservations about this struct (not anything you did) and would love to hear your thoughts on them.

Specifically, because all of these fields are `string` and not `*string`, consumers can't do a partial update of the play without potentially erasing data. They will need to set all of the fields on the struct, or those fields will be zeroed out when the API call is made if they aren't required. 

A side effect of this is that consumers will almost always need to do a Get + Update operation, even if they are only updating a single field to a static value across all of their response plays. 

Should we make these a `*string` and not a `string`, and include `omitempty`, so that folks can do partial updates?

I'm not necessarily asking that we do this, but am instead asking your opinion on it and what you'd maybe do if you were deciding.

Reply:
Thanks @theckman  - yes I think that is a good suggestions, shall I make the changes and update the PR?

Reply:
Sorry for the delay, was noodling on this.

Yeah, let's go with changing these up to be pointer values. :+1: